doc/usage.md

```diff
@@ -166,6 +166,138 @@ $ oc get nodes -w
 Once the nodes reboot, you might want to run another Suite to ensure that
 the remediation that you applied previously was no longer found.
 
+## Evaluating rules against default configuration values
+
+Kubernetes infrastructure may contain incomplete configuration files. At run time, 
+nodes will assume default configuration values for missing configuration options.
+And some configuration can be passed as command line arguments, therefore,
+the Compliance Operator cannot assume the configuration file on the node is complete
+and it may be missing options used in ru le checks.
+
+To prevent false negative findings where the default configuration value passes
+a check, the Compliance Operator generates a complete set of configuration
+values, used specifically for evaluating rules. This increases the accuracy
+of the scan results.
+
+### How to use it?
+
+#### Standard cluster with only `master` and `worker` pool
+In a standard cluster that only has `master` and `worker` pool, there isn't much needed
+to be done, everything will work out of the box.
+
+#### A cluster that uses sub-pools
+
+For example, in a cluster that has `master`, `worker` and `infra` pool, If a user wants 
+to check against all pools, they need to set the value of `ocp-var-role-master` and
+`ocp-var-role-worker` to `infra` in `TailoredProfile`:
+
+```yaml
+apiVersion: compliance.openshift.io/v1alpha1
+kind: TailoredProfile
+metadata:
+  name: cis-infra-tp
+spec:
+  extends: ocp4-cis
+  title: My modified nist profile with a custom value
+  setValues:
+  - name: ocp4-var-role-master
+    value: infra
+    rationale: test for infra nodes
+  - name: ocp4-var-role-worker
+    value: infra
+    rationale: test for infra nodes
+  description: cis-infra-scan
+```
+And launch a scan using `ScanSettingBinding`:
+
+```yaml
+apiVersion: compliance.openshift.io/v1alpha1
+kind: ScanSettingBinding
+metadata:
+  name: cis
+  namespace: openshift-compliance
+profiles:
+- apiGroup: compliance.openshift.io/v1alpha1
+  kind: Profile
+  name: ocp4-cis
+- apiGroup: compliance.openshift.io/v1alpha1
+  kind: Profile
+  name: ocp4-cis-node
+- apiGroup: compliance.openshift.io/v1alpha1
+  kind: TailoredProfile
+  name: cis-infra-tp
+settingsRef:
+  apiGroup: compliance.openshift.io/v1alpha1
+  kind: ScanSetting
+  name: default
+```
+
+#### `KubeletConfig` Remediation on sub-pools
+
+If a user wants to use `KubeletConfig` remediation on sub-pools, they need to add a label
+to the sub-pool[Remediation for Customized MachineConfigPool](https://docs.openshift.com/container-platform/4.11/security/compliance_operator/compliance-operator-remediation.html#compliance-operator-apply-remediation-for-customized-mcp). `MachineConfigPool`: 
```

Comment:
I'm not sure I understand the reason for `MachineConfigPool` starting the sentence?

We do want the oc command though?